packages/angular/src/app/pages/crm-contact-list/crm-contact-list.component.ts

```diff
@@ -36,6 +41,8 @@ type FilterContactStatus = ContactStatus | 'All';
 export class CrmContactListComponent implements OnInit, OnDestroy {
   @ViewChild(DxDataGridComponent, { static: true }) dataGrid: DxDataGridComponent;
 
+  @ViewChild('userPopup', { static: true }) userPopup: FormPopupComponent;
```

Comment:
may be more simple is showing popup by props?

Reply:
In the task list, I am passing the form open method with eventemitter to the kanban component